database/seed.js

```diff
@@ -0,0 +1,41 @@
+const faker = require('faker');
+const request = require('request');
+const db = require('.');
+const fs = require('fs');
```

Comment:
Do you need fs?

Reply:
I do not. I made another commit right after submitting my PR, but it wasn't included in the review.